addons/goldpinger/template/values.yaml

```diff
@@ -0,0 +1,102 @@
+# originally from https://github.com/okgolove/helm-charts/blob/master/charts/goldpinger/values.yaml
+# slightly modified
+
+# Default values for goldpinger.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: bloomberg/goldpinger
+  tag: v3.2.0
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistrKeySecretName
+
+rbac:
+  create: true
+serviceAccount:
+  create: true
+  name: goldpinger # originally "" (replicated)
+
+goldpinger:
+  port: 80
+
+extraEnv:
+  # added block from https://github.com/bloomberg/goldpinger/blob/4f8d872/extras/example-serviceaccounts.yml#L44-L48 (replicated)
+  # podIP is used to select a randomized subset of nodes to ping.
+  - name: POD_IP
+    valueFrom:
+      fieldRef:
+        fieldPath: status.podIP
+
+service:
+  type: ClusterIP # originally "LoadBalancer" (replicated)
+  port: 80
+  annotations: {}
+  labels: {}
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+  # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+  - goldpinger.local
+  labels: {}
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.test
+
+## Set a priorityClassName for the pod. If left blank a default priority will be set.
+priorityClassName:
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits: # enabled block, reduced values (replicated)
+    cpu: 50m
+    memory: 128Mi
+  requests:
+    cpu: 20m
+    memory: 64Mi
+
+podAnnotations: {}
+
+podLabels: {}
+
+## Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+##
+nodeSelector: {}
+
+## Tolerations for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: []
+
+## Affinity for pod assignment
+## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+##
+affinity: {}
+
+## Enable this if pod security policy enabled in your cluster
+## It will bind ServiceAccount with unrestricted podSecurityPolicy
+## Ref: https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+podSecurityPolicy:
+  enabled: false
+  policyName: unrestricted-psp
+
+serviceMonitor:
```

Comment:
Might agree with @emosbaugh to turn this on by default, but wouldn't let it hold up the PR.

Reply:
I would turn this on IF the prom addon is enabled

Reply:
Will do

Reply:
enabled (but not through the helm chart)